src/pages/version/taskDuration/TaskDurationRow.tsx

```diff
@@ -29,12 +30,13 @@ export const TaskDurationRow: React.VFC<RowProps> = forwardRef(
     ref: React.Ref<HTMLTableRowElement>
   ) => {
```

Comment:
I noticed that there's an inner scroll bar on the tasks duration table if there are a lot of display tasks. For example, on any `mongodb-mongo-master` commit I can see a scroll bar in the table, which doesn't happen on the currently deployed version of Spruce. Can check if we can get rid of this duplicate scroll bar? 🤔